### DISCUSSION TITLE
Webhooks for canceled workflows

### DISCUSSION THREAD

**Discussion by Joelasaur** [community/community#29570] [Apps, API and Webhooks] [answered]
I'm reading [the documentation](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#workflow_job) on `workflow_job` webhooks, but I don't see `canceled` as a valid status. What I'm trying to do is send a POST request to Applitools to close the test batch so the job won't hang after the workflow has been canceled. Does such a webhook exist, and if so how do I use it? Thank you.

**Comment by airtower-luna** [comment, marked as answer]
The `workflow_job` element of the payload should have what you need:

> The workflow job. Many workflow_job keys, such as head_sha, conclusion, and started_at are the same as those in a [check_run](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#check_run) object.

If the job has been completed the `check_run` has a `conclusion` element, which tells you *how* it was completed. `cancelled` is one option there.

**Reply by Joelasaur** [reply]
Thank you so much, this is what I was looking for.
